File: Python/program14.py
def main():
    Value = int (input("Enter number"))
    
    Display(Value)
    
def Display(no):
    
    for iCnt in range(no): # will run from 0 to no-1 0-10
        print(iCnt)
    
    
if __name__ == "__main__":
    main()

File: Java/program14.java
import java.util.*;

class DisplayClass
{
    public void Display(int iNo)
    {
        int iCnt = 0;
        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
              System.out.println("Hello");  
        }
     


    }
}

class program14
{
    public static void main(String A[])
    {
        int iValue = 0;
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the frequency :");
        iValue = sobj.nextInt();

        DisplayClass dobj = new DisplayClass();
        dobj.Display(iValue);

        
    }
}